### PR TITLE
[GH-21210] refactor(commands): add error handling to listCommandCmdF

### PR DESCRIPTION
**Pull request by VictorAssunc**
#### Summary
Added error handling in `listCommandCmdF` for errors that were only printed and not returned.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/21210

### CODE REVIEW

**Review by isacikgoz — COMMENTED** [review]
Looks good in general, @VictorAssunc can you address the linter issue?

**Comment by VictorAssunc** [comment]
> 

for sure, i will work on it

**Comment by VictorAssunc** [comment]
done ♻️

File: commands/command.go [hunk 1]
```diff
@@ -195,21 +196,24 @@ func listCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
...
 	for i, team := range teams {
 		if team == nil {
 			printer.PrintError("Unable to find team '" + args[i] + "'")
+			errs = multierror.Append(errs, fmt.Errorf("unable to find team '%s'", args[i]))
 			continue
 		}
 		commands, _, err := c.ListCommands(team.Id, true)
 		if err != nil {
 			printer.PrintError("Unable to list commands for '" + team.Id + "'")
+			errs = multierror.Append(errs, fmt.Errorf("unable to list commands for '%s'", team.Id))
```
Comment (noxer) [inline commands/command.go:209]:
```suggestion
			errs = multierror.Append(errs, fmt.Errorf("unable to list commands for '%s': %w", team.Id, err))
```
We should add the original error here.

**Review by noxer — CHANGES_REQUESTED** [review]
Great work! One small change request from me and we are good to go.

**Comment by VictorAssunc** [comment]
♻️

**Review by noxer — APPROVED** [review]
Looks good, thank you very much @VictorAssunc.

**Review by isacikgoz — APPROVED** [review]

**Pull request merged**